automathor.py

```
@@ -0,0 +1,187 @@
+            logging.debug(f"PROCESSING {line}")
+            data = self.parse_line(line)
+            if data.line_number == self.line:
+                context.commit = data.commit
+                context.author_email = data.author_email
+                context.datetime = data.datetime
+                context.todo = self.text.strip().lstrip("#:TODO ")
+                context.text = self.text.strip().lstrip("#:TODO ")
+            context.lines.append(data)
+        return context
+
+    def parse_line(self, line) -> AnnotatedLine:
```

Ruff (0.8.2)

Missing type annotation for function argument `line`

(ANN001)